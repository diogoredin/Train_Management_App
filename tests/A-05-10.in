4
3
Faro
0
0
