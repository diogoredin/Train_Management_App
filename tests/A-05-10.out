Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Nome da estação: Serviço #182 @ 51.50
07:00 Faro
07:10 Loulé
07:23 Albufeira - Ferreiras
07:29 Tunes
07:59 Santa Clara-Sabóia
08:23 Funcheira
08:42 Ermidas-Sado
08:54 Grândola
09:23 Pinhal Novo
09:57 Lisboa - Entrecampos
10:09 Lisboa - Oriente
11:45 Coimbra-B
12:12 Aveiro
12:39 Vila Nova de Gaia-Devesas
12:44 Porto - Campanhã
Serviço #670 @ 21.20
08:24 Faro
08:35 Loulé
08:47 Albufeira - Ferreiras
08:53 Tunes
09:02 Messines-Alte
09:28 Santa Clara-Sabóia
10:00 Funcheira
10:22 Ermidas-Sado
10:37 Grândola
11:07 Pinhal Novo
11:30 Pragal
11:43 Lisboa - Sete Rios
11:46 Lisboa - Entrecampos
11:56 Lisboa - Oriente
Serviço #672 @ 21.20
13:54 Faro
14:05 Loulé
14:17 Albufeira - Ferreiras
14:23 Tunes
14:31 Messines-Alte
14:57 Santa Clara-Sabóia
15:26 Funcheira
15:47 Ermidas-Sado
16:04 Grândola
16:37 Pinhal Novo
17:04 Pragal
17:14 Lisboa - Sete Rios
17:17 Lisboa - Entrecampos
17:26 Lisboa - Oriente
Serviço #184 @ 51.50
15:05 Faro
15:16 Loulé
15:29 Albufeira - Ferreiras
15:35 Tunes
17:23 Pinhal Novo
17:57 Lisboa - Entrecampos
18:09 Lisboa - Oriente
19:45 Coimbra-B
20:12 Aveiro
20:39 Vila Nova de Gaia-Devesas
20:44 Porto - Campanhã
Serviço #674 @ 21.20
17:56 Faro
18:06 Loulé
18:18 Albufeira - Ferreiras
18:25 Tunes
18:34 Messines-Alte
19:00 Santa Clara-Sabóia
19:29 Funcheira
19:53 Ermidas-Sado
20:08 Grândola
20:40 Pinhal Novo
21:04 Pragal
21:14 Lisboa - Sete Rios
21:17 Lisboa - Entrecampos
21:26 Lisboa - Oriente
Consulta de serviços
1 - Mostrar todos os serviços
2 - Mostrar serviço com um dado número
3 - Mostrar serviços com início em estação
4 - Mostrar serviços com término em estação
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 